include/hpp/core/steering-method/straight.hh

```diff
@@ -21,12 +21,8 @@
 
 # include <hpp/core/config.hh>
 # include <hpp/core/fwd.hh>
-# include <hpp/core/problem.hh>
 # include <hpp/core/steering-method/fwd.hh>
 # include <hpp/core/steering-method.hh>
-# include <hpp/core/straight-path.hh>
-# include <hpp/core/distance.hh>
-# include <hpp/core/config-projector.hh>
```

Comment:
This reduction of header dependency causes a compilation problem in hpp-rbprm. I will provide a fix there.